### PR TITLE
Temporarily hide emails

### PR DESCRIPTION
**Pull request by fatlotus**
I realized on Wednesday that we never really talked about showing people's emails, so I un-showed them, pending further policy discussion.

### CODE REVIEW

File: caravel/templates/listing_form.html [hunk 1]
```diff
@@ -11,18 +11,16 @@
       <div class="col-xs-12">
         <form class="form" method="post" role="form"
             enctype="multipart/form-data">
+          <p>Alumni/BSD/Medicine/Medical School affiliates: if you are unable
```
Comment (georgeteo) [inline caravel/templates/listing_form.html]: Nooo. Why is this back at the top again?

Comment (fatlotus) [inline caravel/templates/listing_form.html]: Oh! I thought that was Git being wonky. Will revert.

**Comment by fatlotus** [comment]
Reverted!

**Comment by georgeteo** [comment]
Has merge conflicts?

File: caravel/templates/listing_form.html [hunk 2]
```diff
@@ -17,12 +17,10 @@
           {% if form.principal %}
             {{ wtf.form_field(form.principal) }}
           {% endif %}
-					<div class="alert alert-warning" role="alert">
-          	BSD/Medicine/Medical School affiliates: if you are unable to sign
-            in with your CNetID, please enter your email in the box. Your
-            listing might not be posted immediately. We reserve the right to
-            remove listings at any time.
-					</div>
+          <p>Alumni/BSD/Medicine/Medical School affiliates: if you are unable
```
Comment (georgeteo) [inline caravel/templates/listing_form.html]: Why is it a <p> tag instead of the <div alert>?

Comment (fatlotus) [inline caravel/templates/listing_form.html]: Oh, good point. I think I must've been using an old `master` for this one...

File: caravel/templates/listing_form.html [hunk 3]
```diff
@@ -17,12 +17,11 @@
           {% if form.principal %}
             {{ wtf.form_field(form.principal) }}
           {% endif %}
-					<div class="alert alert-warning" role="alert">
-          	BSD/Medicine/Medical School affiliates: if you are unable to sign
-            in with your CNetID, please enter your email in the box. Your
-            listing might not be posted immediately. We reserve the right to
-            remove listings at any time.
-					</div>
+          <p class="alert alert-warning" role="alert">
```
Comment (georgeteo) [inline caravel/templates/listing_form.html]: Have not reverted to the correct master?

Comment (fatlotus) [inline caravel/templates/listing_form.html]: I think it is. The `<p>` is more semantic than a `<div>`, but it should show up in the same manner.

Comment (georgeteo) [inline caravel/templates/listing_form.html]: Oh oops. I missed the CSS class there. Sure.

**Pull request merged**